### DISCUSSION TITLE
Azure Enterprise Policy as Code vs AzOps

### DISCUSSION THREAD

**Discussion by ri-we** [Azure/AzOps#681] [Q&A] [answered]
It looks like some Microsoft FTE started a new policy as code initiative, must say its a bit confusing for my customer https://techcommunity.microsoft.com/t5/core-infrastructure-and-security/azure-enterprise-policy-as-code-a-new-approach/ba-p/3607843

What is the AzOps view on this? My customer is doubting about AzOps and this doesn't help.

We use AzOps mainly for policy deployment and management groups.

**Comment by daltondhcp** [comment, marked as answer]
Thank you for your question! 

As stated in the beginning of the article mentioned above, it represents an **alternative approach and views based on the authors opinions** and not MSFT as a whole. Depending on architecture and requirements for policy lifecycle management, it might be that it is a good fit in certain scenarios.

With AzOps, we take a slightly different approach with a generic pipeline that can be used across all Azure scopes regardless of resource type, not only policies.
Since we also support reconciling/managing resources created with a non-IaC methods into git as well as full-on code-first approaches, we believe can meet the customers where they are in terms of maturity and operating model. 

What works best for you and your customers will of course depend on what the requirements and operating model preferences are.
